### PR TITLE
Fix tests.foreman.api.test_subscription.SubscriptionsTestCase.test_negative_create_1

### LINKED ISSUE #2167: Automation failure : test_negative_create_1
TestNames:
tests.foreman.api.test_subscription.SubscriptionsTestCase.test_negative_create_1

the test is failing due to : 
Task b1e723fc-03ac-4948-a301-9e7879be78e9 completed with result warning. Error message(s): [u'This subscription management application has already been imported by another owner.']

It seems that the subscription used should be unique . please fix this
thanks

### PR DESCRIPTION
**Pull request by oshtaier**
Fix tests corresponding to a new exception handling approach after code migration to nailgun

Closes #2167

### CODE REVIEW

**Comment by oshtaier** [comment]
```
nosetests tests/foreman/api/test_subscription.py -m test_negative_create_1
.
----------------------------------------------------------------------
Ran 1 test in 83.537s

OK
```

File: tests/foreman/api/test_subscription.py [hunk 1]
```diff
@@ -52,12 +53,10 @@ def test_negative_create_1(self):
...
 
-        # Upload the manifest to a second organization.
-        org = entities.Organization()
-        org.id = org.create_json()['id']
-        org.upload_manifest(manifest_path)
-        self.assertEqual([], org.subscriptions())
+        second_org = entities.Organization().create()
+        with self.assertRaises(TaskFailedError):
+            second_org.upload_manifest(manifest_path)
+        self.assertEqual([], second_org.subscriptions())
```
Comment (elyezer) [inline tests/foreman/api/test_subscription.py]: Or `self.assertEqual(len(second_org.subscriptions()), 0)`

**Comment by elyezer** [comment]
Looks good ACK.

**Comment by jyejare** [comment]
Good. ACK

**Comment by sthirugn** [comment]
ACK

**Comment by Ichimonji10** [comment]
:+1:

**Pull request merged**